src/common/main.lua

```diff
@@ -54,6 +64,41 @@ main.init = function()
 			Data = json.encode(utils.affiliationsForAddress(address, ANTS)),
 		})
 	end)
+
+	utils.createActionHandler(ActionMap.AddVersion, function(msg)
+		assert(msg.From == Owner, "Only ANT Registry owner can add versions")
+		local version = msg.Version
+		local module = msg["Module-Id"]
+		local luaSource = msg["Lua-Source-Id"]
+
+		utils.validateSemver(version)
+		utils.validateArweaveId(module)
+		assert(
+			type(luaSource) == "string" and utils.validateArweaveId(luaSource) or luaSource == nil,
+			"Lua-Source-Id should be a valid arweave ID"
+		)
+		assert(not ANTVersions[version], "Version " .. tostring(version) .. " already exists")
```

Comment:
We might want to consider relaxing this constraint and making this handler more of an upsert.

Reply:
Like to replace the ID's on an existing version?